ftw/tagging/browser/configure.zcml

```diff
@@ -5,7 +5,6 @@
     <browser:resourceDirectory
         name="ftw.tagging.stylesheets"
         directory="stylesheets"
-        layer=".interfaces.IFtwTaggingLayer"
```

Comment:
Why did you remove this?

Reply:
resourceDirectory does not support a layer.